dodoc/config.ts

```diff
@@ -201,11 +201,25 @@ const formatCode = (code: string, type: string) => {
     .trim();
 
   if (!formatedCode.endsWith('()')) {
+    const start = `${formatedCode.split('(')[0]}(`;
+    const end = ')';
+    const middle = formatedCode.replace(start, '').replace(end, '');
```

Comment:
typo in variable name

```suggestion
    const middle = formattedCode.replace(start, '').replace(end, '');
```